contingency/contingency-api/src/main/java/com/powsybl/contingency/tasks/AbstractInjectionTripping.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.contingency.tasks;
+
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.network.*;
+
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * @author Teofil Calin BANC <teofil-calin.banc at rte-france.com>
+ */
+public abstract class AbstractInjectionTripping extends AbstractTrippingTask {
```

Comment:
Should this class be generic?

Reply:
If it's OK with you : after discussing with @calinbanc we decided that it does not appear useful enough.
We can even see it this way: having it generic could mean that the actual type matters, whereas here we really only need an `Injection`, nothing more.
It would have been more useful if it allowed to not have subclasses, but we still need them anyway.